tests/cli/demultiplex/test_demultiplex_flowcell.py

```diff
@@ -228,15 +216,12 @@ def test_start_demultiplex_flow_cell(
         obj=demultiplex_context,
     )
 
+    # THEN the demultiplexing should have started
+    assert "Creating demultiplexing started file" in caplog.text
+
     # THEN assert the command exits without problems
     assert result.exit_code == 0
 
-    # THEN assert it was communicated that previous demux was deleted
-    assert f"Removing flow cell demultiplexing directory {bcl2fastq_flow_cell.path}"

```

Comment:
whaaa - this code is mind-blowing 🤯

Reply:
I dont think it has a lot of value - or if the test is actually mature enough - but I guess it would be nice, since the path is touched above, to see that the path was also removed?

Reply:
I guess I am suggesting to do an assert that the path is actually removed when the command is invoked - I think that's what it was "supposed" to be doing

Reply:
I don't see what we're testing, and the CLI command is tested before, so imma just delete it